plugins/tests/sqlalchemy/test_task.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+import tempfile
+import time
+from subprocess import Popen
+
+import doltcli as dolt
+import pandas
+import pytest
+from flytekitplugins.sqlalchemy import SQLAlchemyConfig, SQLAlchemyTask
+
+from flytekit import Secret, current_context, kwtypes, task, workflow
+from flytekit.types.schema import FlyteSchema
+
+OK_EXAMPLE_DB = "mysql+pymysql://root@localhost:3307/tracks"
+BAD_EXAMPLE_DB = "mysql+pymysql://root1@localhost/tracks"
+
+# This task belongs to test_task_static but is intentionally here to help test tracking
+tk = SQLAlchemyTask(
+    "test",
+    query_template="select * from tracks",
+    task_config=SQLAlchemyConfig(
+        uri=OK_EXAMPLE_DB,
+    ),
+)
+
+
+@pytest.fixture(scope="function")
+def sql_server():
+    p = None
+    try:
+        d = tempfile.TemporaryDirectory()
+        db_path = os.path.join(d.name, "tracks")
+        db = dolt.Dolt.init(db_path)
```

Comment:
Could we just remove this test, and then remove all dolt-specific dependencies?  Then we should merge.  I'm totally happy having a dolt-specific example in the flytesnacks repository under [cookbook/plugins](https://github.com/flyteorg/flytesnacks/tree/master/cookbook/plugins) that uses this task.  I think that would be cleanest.